tests/integration_tests/test_filters.py

```diff
@@ -1,13 +1,58 @@
+from pathlib import Path
+
 import pytest
 from web3 import Web3
 
+from .network import setup_custom_ethermint, setup_ethermint
 from .utils import (
     ADDRS,
     CONTRACTS,
     deploy_contract,
     send_successful_transaction,
     send_transaction,
+    w3_wait_for_new_blocks,
+)
+
+
+@pytest.fixture(scope="module")
+def custom_ethermint(tmp_path_factory):
+    path = tmp_path_factory.mktemp("filters")
+    yield from setup_ethermint(path, 26200, long_timeout_commit=True)
+
+
+@pytest.fixture(scope="module")
+def ethermint_indexer(tmp_path_factory):
+    path = tmp_path_factory.mktemp("indexer")
+    yield from setup_custom_ethermint(
+        path, 26660, Path(__file__).parent / "configs/enable-indexer.jsonnet"
+    )
+
+
+@pytest.fixture(
+    scope="module", params=["ethermint", "geth", "ethermint-ws", "enable-indexer"]
 )
+def cluster(request, custom_ethermint, ethermint_indexer, geth):
```

Comment:
I need limit to module scope to reduce zombie supervisor process left when unexpected exit status 1, any better idea? @yihuang